index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Drag Race Quest</title>
+    <link rel="stylesheet" href="./src/main.css">
+</head>
+<body>
+    <header>
+        <h1>RuPaul's Drag Quest</h1>
+    </header>
+    <main>
+        <h2>Start Your Engines</h2>
+        <form id="user-sign-up">
+            <p class="text-field">
+                <label for="name">Your Drag Name:</label>
+                <input id="name" name="name" required>
+            </p>
+            <fieldset>
+                <legend >Pick Your Queen:
+                    <p>
+                        <label>
```

Comment:
on line 18 you have ```<label for="name">``` but then starting on line 24 you are wrapping your ```<input>``` elements in ```<label>``` tags.